### PR TITLE
Patch for #82 after test in async regime

### PR DESCRIPTION
**Pull request by vvchernov**
Small fixes after testing in async regime on mlc-llm side

### CODE REVIEW

File: serve/mlc_serve/utils.py [hunk 1]
```diff
@@ -25,7 +25,6 @@ def get_default_mlc_serve_argparser(description="", allow_override=False):
     parser.add_argument("--use-sync-engine", action="store_true")
     parser.add_argument("--num-sequences-to-sample", type=int, default=1)
     parser.add_argument("--max-num-batched-tokens", type=int, default=4096)
-    parser.add_argument("--num-sequences-to-sample", type=int, default=1)
```
Comment (zxybazh) [inline serve/mlc_serve/utils.py:28]: Hi, is there any specific reason for this removal?

**Review by zxybazh — COMMENTED** [review]
Otherwise LGTM.

File: serve/mlc_serve/utils.py (hunk 1 above)
Comment (vvchernov) [reply serve/mlc_serve/utils.py:28]: Yes, it is duplicate of line 26

Comment (zxybazh) [reply serve/mlc_serve/utils.py:28]: Got it.

**Pull request merged**